srcpkgs/Slic3rPE/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'Slic3rPE'
+pkgname=Slic3rPE
+version=1.42.0
+revision=1
+_status="-alpha6"
```

Comment:
alpha software ?

Reply:
Please see the original description I wrote